hooks/useBlockBySeq.tsx

```diff
@@ -0,0 +1,18 @@
+import { useContext, useEffect } from "react"
+
+import { BlockContext } from "contexts/ServiceContexts"
+import { BlockType } from "types"
+import useAsyncDataWrapper from "./useAsyncDataWrapper"
+
+const useBlockBySeq = (seq: number) => {
+  const service = useContext(BlockContext)
+  const [result, wrapper] = useAsyncDataWrapper<BlockType>()
+
+  useEffect(() => {
+    wrapper(service.find({ sequence: seq }))
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What specifically isn't being included in deps here? `service` && `wrapper`? Is this disabling required?

Reply:
Yes, it is required. No need to reload block in case if hook get new instance of `service` or `wrapper`